dump/src/test/resources/shaclTestsCoverageTable.md

```
@@ -26,10 +26,10 @@ wikipage-uri|shacl-test|issue|comment
 [http://cv.dbpedia.org/resource/Берлин](http://dief.tools.dbpedia.org/server/extraction/cv/extract?title=Берлин&revid=&format=trix&extractors=custom) | 
 [http://cy.dbpedia.org/resource/Berlin](http://dief.tools.dbpedia.org/server/extraction/cy/extract?title=Berlin&revid=&format=trix&extractors=custom) | 
 [http://da.dbpedia.org/resource/Berlin](http://dief.tools.dbpedia.org/server/extraction/da/extract?title=Berlin&revid=&format=trix&extractors=custom) | 
-[http://de.dbpedia.org/resource/Arthur_Schopenhauer](http://dief.tools.dbpedia.org/server/extraction/de/extract?title=Arthur_Schopenhauer&revid=&format=trix&extractors=custom) | [http://dbpedia.org/ontology/abstract](http://dbpedia.org/ontology/abstract) #en_abstract_validation | 
-[http://de.dbpedia.org/resource/Berlin](http://dief.tools.dbpedia.org/server/extraction/de/extract?title=Berlin&revid=&format=trix&extractors=custom) | [http://dbpedia.org/ontology/abstract](http://dbpedia.org/ontology/abstract) #en_abstract_validation | 
+[http://de.dbpedia.org/resource/Arthur_Schopenhauer](http://dief.tools.dbpedia.org/server/extraction/de/extract?title=Arthur_Schopenhauer&revid=&format=trix&extractors=custom) | 
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)